src/components/SVG/Delete.js

```diff
@@ -0,0 +1,22 @@
+const DeleteIcon = ({ options, setInitialData }) => {
+  const { width, height } = options;
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width={width}
+      height={height}
+      fill="currentColor"
+      className="bi bi-trash main-icons"
+      viewBox="0 0 16 16"
+      onClick={() => setInitialData()}
```

Comment:
and why is it necessary to broadcast a click on svg, and not on div, which is higher?